README.md

```diff
@@ -105,19 +118,20 @@ STATIC_URL = '/static/'
 STATICFILES_DIRS = (
   '‘/usr/local/devcycle-server/tour_config/static/’
   )
-  
-Restart the apache server to put all changes into effect. 
+
+Restart the apache server to put all changes into effect.
 
 ```
-/etc/init.d/apache2 reload
+sudo /etc/init.d/apache2 reload
 ```
 
-###Migrate the Database Schema using South
+
+###Migrate the Database Schema using South (Setup The Database)
```

Comment:
When I got to this point, I wasn't sure if I should continue. I haven't set up the database yet, why would I migrate it?

I suggest calling this section just "Setup the Database" since it looks like we're using South to create it. rather than to migrate it. At the end of the section we could put the first part of the section about the migration.